examples/grf/sim_ludo_analysis.jl

```diff
@@ -0,0 +1,108 @@
+#=
+This solves the equation
+du = (∇^2 - β) tanh(γu) - α + σ dW
+for a 2D image u, of size NxN, assuming
+- Dirichlet periodic boundary conditions
+half indices indicate faces
+- Δx = Δy = 1
+- <dW(x1,y1,t) dW(x2,y2,s)> = δ(t-s)[(x1-x2)^2 + (y1-y2)^2 + 1]^(-1/2)
+- σ, α, β, γ are scalars
+=#
+using LinearAlgebra
+using ProgressBars
+using Random
+using StatsBase
+include("./utils.jl")
+include("./rhs.jl")
+# Set up model
+FT = Float32
+# Image is NxN
+N = 8
+α = FT(3)
+β = FT(0.1)
+γ = FT(20)
+σ = FT(2.0)
+
+# Boundary condition - periodic
+Γ = FT.(reshape(zeros(N^4), (N^2,N^2)))
+for i1 in 1:N
+    for j1 in 1:N
+        k1 = (j1-1)*N+i1
+        for i2 in 1:N
+            for j2 in 1:N
+                k2 = (j2-1)*N+i2
+                Γ[k1,k2] = FT(1.0/sqrt((i1-i2)^2 + (j1-j2)^2+1))
```

Comment:
double check this is the same as what Ludo used?

Reply:
Modified to Ludo's code